bugwarrior/db.py

```diff
@@ -89,7 +95,7 @@ def hamdist(str1, str2):
 
 def get_managed_task_uuids(tw, key_list, legacy_matching):
     expected_task_ids = set([])
-    for keys in key_list.values():
+    for keys in list(key_list.values()):
```

Comment:
Because `values()` in py3 is a view and not an iterator.   The following code work as expected in py2 while it raise a RuntimeError in py3.4:
```
a = {1: 1, 2: 2, 3: 3}
for i in a.values():
     a[-1] = 1
```
Maybe today the bw code does not expose this effect, but without `list()` tomorrow an innocent change may break something.

this is a change introduced by futurize to *ensure*, IMHO, the same behavior in both version.

If you think `list()` should still be removed I'll work on it.